test/api/buffer/import/import/expected-etcd-kvs.txt

```diff
@@ -185,7 +185,7 @@
   "fileId": "%s",
   "sliceId": "%s",
   "count": 2,
-  "size": 567,
+  "size": %d,
```

Comment:
Z nejakeho dovodu dostavam lokalne IP adresu `127.0.0.1`, a v CI `::1` (IPv6), teda `size` moze mat roznu hodnotu.

Reply:
Na to už jsem taky při něčem nedávno narazil. 🙂